src/chrome/content/rules/Bayern.de.xml

```diff
@@ -0,0 +1,27 @@
+<!-- cert mismatch on:
+	- www.ldbv.bayern.de
+	- www.gdi.bayern.de/
+
+	intergeo domains outside of the 10-16 range do not support TLS
+-->
+<ruleset name="Bayern.de">
```

Comment:
Please mark the rule as `partial` (depending on how many securable/non-working domains you will find).